### DISCUSSION TITLE
Incompatible version

### DISCUSSION THREAD

**Discussion by avelopolcak** [Digitalroot-Valheim/Digitalroot.Valheim.MaxDungeonRooms#22] [Q&A] [answered]
My friend and I are trying to get a modded server working, but it seems like it's impossible to get it to work. Both local and through dedicated server. When he tries to connect through a dedicated server we get this error:

[Info   : Unity Log] 01/29/2023 20:14:29: Player joined server "RockandStone" that has join code 041646, now 2 player(s)

[Info   : Unity Log] 01/29/2023 20:14:29: Muted PlayFab remote player F2717E6F377C0006

[Info   : Unity Log] 01/29/2023 20:14:29: PlayFab socket with remote ID playfab/F2717E6F377C0006 received local Platform ID Steam_76561198045758755

[Info   : Unity Log] 01/29/2023 20:14:29: ZRpc timeout set to 90s 

[Info   : Unity Log] Sending Epic Loot version 0.9.8 and minimum version 0.9.8 to the client.
[Info   : Unity Log] 01/29/2023 20:14:29: Got handshake from client playfab/F2717E6F377C0006

[Warning:Jotunn.Utils.ModCompatibility] Missing mod on client: Digitalroot Max Dungeon Rooms
[Warning:Jotunn.Utils.ModCompatibility] RPC_Jotunn_ReceiveVersionData: Disconnecting modded client with incompatible version message. Mods are not compatible
[Info   : Unity Log] 01/29/2023 20:14:30: RPC_Disconnect 

[Info   : Unity Log] 01/29/2023 20:14:30: Player connection lost server "RockandStone" that has join code 041646, now 1 player(s)

[Info   : Unity Log] 01/29/2023 20:16:14: Keep socket for playfab/8F86C4B306B8C329, try to reconnect before timeout

[Info   : Unity Log] 01/29/2023 20:16:14: Player connection lost server "RockandStone" that has join code 041646, now 1 player(s)

We both have the same version of the mod installed

**Comment by Digitalroot** [comment, marked as answer]
If you remove the mod are they able to connect?

**Reply by avelopolcak** [reply]
He gets an incompatible version message with Jotunn Library also, but before that he was able to connect. Same version of Jotunn as well

**Reply by Digitalroot** [reply]
Sounds like there's a general problem, and not really a problem specific to the mod. I suggest asking around in the modding discords and see if you can get someone to look at your log files and try to help you troubleshoot the issue.
